osmtm/models.py

```diff
@@ -372,6 +376,18 @@ def to_feature(self):
             properties=properties
         )
 
+    def get_extra_instructions(self):
+        instructions = self.project.per_task_instructions
+        properties = {}
+        if self.x:
+            properties['x'] = str(self.x)
+        if self.y:
+            properties['y'] = str(self.y)
+        if self.zoom:
+            properties['z'] = str(self.zoom)
+        properties.update(_loads(self.extra_properties))
```

Comment:
This should probably be in a try or if statement since it is undefined by L346-347 if no extra properties exist.